### PR TITLE
Add 'Contributors' Guide' section to /docs/

### PR DESCRIPTION
**Pull request by wjdp**
Re #754. Targeted at new contributors.

Removed GH CONTRIBUTING.md, moved to .github.

Tweaks to docs styling made during writing.

Please review my writing!

### TODO

- [x] /docs/contributing (new page)
- [x] /docs/editing (needs rewriting as editing intro)

### CODE REVIEW

**Comment by wjdp** [comment]
Also suggest as part of this we put the majority (if not all) changes through PRs ( @nhyty :)

**Comment by davidtaylorhq** [comment]
Yep, I think doing everything through PRs is sensible, probably default to squash merges

File: _content/docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+The process of getting a change committed and a pull request opened may seem daunting at first, but stick with it. Once you've done it the first time it's not too bad, honest!
+
+#### Forking and branching
+
+Firstly you'll need to create your own _fork_ of the respository on which to work. Head over to the [repository][gh-repo] and hit the _<i class="octicon octicon-repo-forked"></i> Fork_ button up top-right. You'll need to create a free GitHub account if you don't have one already.
+
+#### Using the web interface (github.com)
+
+1. Find the file you want to edit. Shows are stored in the `_shows` folder, people in `_people` _e.t.c._ See the [editing documentation](/docs/editing/) for more information on actually making your edit. The _Find file_ button is quite useful for this.
```
Comment (samozzy) [inline _content/docs/contributing.md]: Would suggest `_e.t.c._` is simply `etc.`

File: _content/docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+{: .box-info}
+
+1. Fork the project by clicking "Fork" in the top right corner of [newtheatre/history-project](https://github.com/newtheatre/history-project).
+2. Clone the repository locally `git clone https://github.com/<your-username>/history-project`.
+3. Create a new, descriptively named branch to contain your change `git checkout -b my-awesome-change`.
+4. Using your preferred editor, make your changes.
+
+#### Writing a good commit message
+
+Using git we can see _what_ changed in a particular commit, only a good commit message can tell us _why_. This is very important for the future when other editors will be updating your work. In the case of show data, they'll need to know your sources and how credible information you've gathered is.
```
Comment (samozzy) [inline _content/docs/contributing.md]: Perhaps replace `only a good` with `but a good`?

File: _content/docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+When writing commits, try and adhere to the following rules:
+
+1. Limit the subject line to **about 50 characters**.
+2. **Capitalize** the subject line and do not end it with a period.
```
Comment (samozzy) [inline _content/docs/contributing.md]: Replace `period` with `full stop`. We're not Americans.

**Comment by nhyty** [comment]
@samozzy and @wjdp , you now happy for this to be merged in???

**Comment by wjdp** [comment]
Nope, see the to-do list in the description. /docs/editing needs a rewrite too

File: _content/docs/contributing.md [hunk 4]
```diff
@@ -62,12 +62,12 @@ This method requires a certain amount of technical knowledge but is a lot more f
 
 #### Writing a good commit message
 
-Using git we can see _what_ changed in a particular commit, only a good commit message can tell us _why_. This is very important for the future when other editors will be updating your work. In the case of show data, they'll need to know your sources and how credible information you've gathered is.
+Using git we can see _what_ changed in a particular commit, but a good commit message can tell us _why_. This is very important for the future when other editors will be updating your work. In the case of show data, they'll need to know your sources and how credible the information you've gathered is.
```
Comment (wjdp) [inline _content/docs/contributing.md]: think this needs to be `but only`. the commit and the message are different things.

File: _content/docs/contributing.md [hunk 5]
```diff
@@ -62,12 +62,12 @@ This method requires a certain amount of technical knowledge but is a lot more f
...
 
 When writing commits, try and adhere to the following rules:
 
 1. Limit the subject line to **about 50 characters**.
-2. **Capitalize** the subject line and do not end it with a period.
+2. **Capitalize** the subject line and do not end it with a full stop.
```
Comment (wjdp) [inline _content/docs/contributing.md]: D-oh. I'm a twit, too much copypasta.

**Comment by nhyty** [comment]
What sort of thing do you want on the editing page @wjdp

**Comment by wjdp** [comment]
The contributing page is all about deciding what to work on and what to do to commit and make a PR but only says this about actually making changes:  `Using your preferred editor, make your changes` 

The editing page should serve as an introduction to 'how to edit'

- Where things are in the directory structure
- Overview of YAML frontmattered pages, perhaps find and quote/link from Jekyll docs
- Perhaps lean into / combine with the style guide issues #750 
- Touch on photos / videos / binary storage

There are all the pages beneath which are detailed docs about each section, it should be an intro for that.

**Comment by nhyty** [comment]
@samozzy or @BenWoodford76 , are either of you happy to make this a thing???

**Comment by BenWoodford76** [comment]
@nhyty I'll have a look at it over the next few days

**Comment by samozzy** [comment]
Me too.

**Comment by BenWoodford76** [comment]
are we looking at both the editing and contributing pages??

**Comment by wjdp** [comment]
Think I mostly did the contributing page, should just be editing one.

**Comment by BenWoodford76** [comment]
@wjdp Cool, we'll check contributing for spelling and grammar

**Comment by BenWoodford76** [comment]
I've been fairly busy and haven't really been able to look at this much. @samozzy now you're done at the Fringe if you get bored can you have a look and see what can be done?

**Comment by BenWoodford76** [comment]
Will get bored and look at this on Sunday perhaps

**Comment by BenWoodford76** [comment]
@samozzy or @nhyty can you look at the bits I'm not comfortable writing stuff about please? :)

**Comment by nhyty** [comment]
which bits?

File: _content/docs/contributing.md [hunk 6]
```diff
@@ -58,7 +58,7 @@ This method requires a certain amount of technical knowledge but is a lot more f
 1. Fork the project by clicking "Fork" in the top right corner of [newtheatre/history-project](https://github.com/newtheatre/history-project).
 2. Clone the repository locally `git clone https://github.com/<your-username>/history-project`.
 3. Create a new, descriptively named branch to contain your change `git checkout -b my-awesome-change`.
-4. Using your preferred editor, make your changes.
+4. Using your preferred editor, make your changes. See the editing page for more detailed information on this.
```
Comment (wjdp) [inline _content/docs/contributing.md]: Can we link "editing page" to the described target?

File: _content/docs/editing.md [hunk 7]
```diff
@@ -39,3 +39,33 @@ Getting the site running on your machine locally is easy to do on Mac and Linux.
 A combination of HTML and Markdown (Kramdown variant) is used for marking up our pages. YAML is used for meta data and is documented for shows, committees etc here.
 
 The [Kramdown syntax guide](http://kramdown.gettalong.org/syntax.html) is useful for reference.
+
+## Where Things Live - Basic Editing
+
+When first looking at the repository in a text editor, it can seem quite big, and finding where things live will take a bit of time, yet most of the content storage itself is self explanatory. For a basic contributor the main things you'll need to find are person files, show files and venue files. Individual person files live in the people folder from the main History Project repository. For info on hwo to format a person file, see the [People](https://history.newtheatre.org.uk/docs/person/) page. Files for individual shows live in the shows folder and are sorted by academic year, see the [Shows](https://history.newtheatre.org.uk/docs/show/). Venue files live in the venues folder, see the [Venues](https://history.newtheatre.org.uk/docs/venue/) page.
```
Comment (wjdp) [inline _content/docs/editing.md]: "hwo" → "how"

Shows **page**

can we refer to directories like this `_people/` (including the 'code' type formatting)

File: _content/docs/editing.md [hunk 8]
```diff
@@ -39,3 +39,33 @@ Getting the site running on your machine locally is easy to do on Mac and Linux.
...
+
+## Years ##
+
+Years are one of the main attributes that are used within the project. These are important to get right and important for many different reasons. For more information on the correct formatting and usage, see the [Years](https://history.newtheatre.org.uk/docs/year/) page.
+
+## Committees ##
+
+New committees are usually added by an [owner](https://github.com/orgs/newtheatre/people) at the start of each academic year. Committee files are stored in the committees folder. We do get submissions from alumni with information regarding past committees and these can be added by anyone. For detailed information on how to format a committee file, see the [Committees](https://history.newtheatre.org.uk/docs/committee/) page.
```
Comment (wjdp) [inline _content/docs/editing.md]: In project parlance this is an editor.

File: _content/docs/editing.md [hunk 9]
```diff
@@ -39,3 +39,33 @@ Getting the site running on your machine locally is easy to do on Mac and Linux.
...
+
+These are used particularly for reviews placed within show files, but the list of possible links is expanding as we grow the site, so could change in the future. For formatting details see the [Link List](https://history.newtheatre.org.uk/docs/link-list/) page.
+
+## Trivia ##
+
+One of the things we love collecting for the History Project are little bits of trivia or anecdotes regarding shows. Be this a last minute cast change for one night or something that happened that amused the cast, these stories are a vital part of what makes the NNT the NNT. To find out how to format these sections, visit the [Trivia List](https://history.newtheatre.org.uk/docs/trivia-list/) page.
+
+## Photos and Assets ##
+
+The History projects visual records are stored in a [SmugMug Site](https://photos.newtheatre.org.uk/). Anyone can view the production shots from any of our shows, but the other archives are password protected. Any show videos, such as Trailers or Behind the Scenes videos are also stored here. If you have any items you feel would be useful to the History Project, please see the [Sending In Images](https://history.newtheatre.org.uk/upload/)page.
```
Comment (wjdp) [inline _content/docs/editing.md]: Technically not password protected, just not advertised publicly. If you have the URL to an album you can view it. Only reason we did this was to clean up the home page of the `photos.new…` site.

File: _content/docs/editing.md (hunk 7 above)
Comment (BenWoodford76) [reply _content/docs/editing.md]: How do you format that correctly?? I either didn't know or have forgotten

**Comment by BenWoodford76** [comment]
@samozzy if you get bored this weekend feel free to have a go at the two bits in the list above I don't feel confident writing about

**Comment by samozzy** [comment]
@BenWoodford76 Won’t happen this weekend, but potentially end of next week job.

File: _content/docs/editing.md [hunk 10]
```diff
@@ -42,15 +42,15 @@ The [Kramdown syntax guide](http://kramdown.gettalong.org/syntax.html) is useful
...
 
 ## Years ##
 
 Years are one of the main attributes that are used within the project. These are important to get right and important for many different reasons. For more information on the correct formatting and usage, see the [Years](https://history.newtheatre.org.uk/docs/year/) page.
 
 ## Committees ##
 
-New committees are usually added by an [owner](https://github.com/orgs/newtheatre/people) at the start of each academic year. Committee files are stored in the committees folder. We do get submissions from alumni with information regarding past committees and these can be added by anyone. For detailed information on how to format a committee file, see the [Committees](https://history.newtheatre.org.uk/docs/committee/) page.
```
Comment (nhyty) [inline _content/docs/editing.md]: a not an in front of project editor

**Comment by wjdp** [comment]
image

Not really sure what's going on here but this branch is going to need some surgery.

**Comment by nhyty** [comment]
@samozzy, any chance you could look at this?

**Comment by samozzy** [comment]
@nhyty Anything specific or just the whole lot?

**Comment by nhyty** [comment]
@samozzy the whole lot but esp the bits ben hasnt done

**Comment by BenWoodford76** [comment]
@nhyty Please feel free to check the SPAG on my bits @samozzy

**Comment by nhyty** [comment]
@samozzy, ever get a chance to do this? specifically the editing page pls 😄

**Comment by samozzy** [comment]
Have given both `contributing.md` and `editing.md` a thorough edit this evening. Please let me know what you think and if I've gone too deep/not deep enough!

File: _content/docs/editing.md [hunk 11]
```diff
@@ -1,38 +1,114 @@
...
+
+You can learn more about how Jekyll's templates work by reading [their documentation](https://jekyllrb.com/docs/templates/)
+
+Our CSS files are compiled using Sass to create a consistent look across all of our pages, and we have a number of Javascript functions running throughout. 
+
+### Advanced Functionality - Ruby
+
+Marrying up a vast amount of data with static templates is no mean feat, and sometimes this data needs to be manipulated first. We do that with Ruby. 
+
+We've also got some Ruby plugins written for Jekyll on the side which allow us to, for instance, see people's commitee positions on their profiles.
```
Comment (BenWoodford76) [inline _content/docs/editing.md]: Do you mean side or site?

Comment (samozzy) [reply _content/docs/editing.md]: Site, oop. Thanks.

**Comment by BenWoodford76** [comment]
Dear god Rebasing is not fun

**Comment by BenWoodford76** [comment]
@BenWoodford76 Once this has passed and if the deploy preview looks good are people happy to merge??

@wjdp0 @nhyty ??

**Comment by wjdp** [comment]
Merged in via #1158

**Pull request closed without merging**